rules.ini

```
@@ -109,6 +109,7 @@ Pico8 = (?:^|/)data\.pod$
 PlayFirstPlayground = \.pfp$
 Prism3D[] = (?:^|/)base\.scs$
 Prism3D[] = (?:^|/)p3shared\.dll$
+RealLive = (?:^|/)(?:RealLiveSteam|rl_d3d)\.dll
```

```suggestion
RealLive = (?:^|/)rl_d3d\.dll
```
There's only 3 game in total